### PR TITLE
see description

### PR DESCRIPTION
**Pull request by xe-rx**
added a pytest for convert.py, see docstring of file. added conftest.py in root of /tests directory with fixtures used.

### CODE REVIEW

File: db/tests/unit/db/models/test_convert.py
Comment (JoelvanIngen) [inline db/tests/unit/db/models/test_convert.py:1]: Please also include:
- Symmetry tests `submission_post_to_db_submission(db_submission_to_submission_get(entry)) == entry`
- Crash tests (are expected errors raised on non-existing/missing variables, are there any edge-case variables that might violate symmetry or crash any of the functions)

File: db/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    UserGet,
+    SubmissionPost,
+    SubmissionGet,
+    ProblemGet,
+)
+
+""""
+I've written fixtures for all three (current) possible schemas that
+src/db/models/convert.py could ecounter.
+"""
```
Comment (JoelvanIngen) [inline db/tests/conftest.py:17]: Status/progress update should be a github comment instead of a code docstring

File: db/tests/conftest.py
Comment (JoelvanIngen) [inline db/tests/conftest.py:1]: Some fixtures are called "sample", some "expected". I think following existing convention and calling them

```
@pytest.fixture(name="[name]")
def [name]_prefix()
```

would be clearest

**Review by JoelvanIngen — CHANGES_REQUESTED** [review]

File: db/tests/unit/db/models/test_convert.py
Comment (xe-rx) [reply db/tests/unit/db/models/test_convert.py:1]: Ik zie geen schemas waarvoor ik een symmetry test zou kunnen/moeten schrijven

Comment (JoelvanIngen) [reply db/tests/unit/db/models/test_convert.py:1]: Like my example; convert something, convert it back, and check you've got the same thing back as you inputted

Comment (JoelvanIngen) [reply db/tests/unit/db/models/test_convert.py:1]: Don't bother for now, we won't spend more time on tests

**Comment by JoelvanIngen** [comment]
Could you fix the errors so we can merge this
